debs/SPECS/wazuh-agent/debian/changelog

```diff
@@ -1,3 +1,9 @@
+wazuh-agent (4.4.3-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
```

Comment:
If you use current then this gets obsolete very fast.

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/2222/commits/5b9fbc36f9ff5e21d9cf38ff13eca28bc7aee0ca